.github/workflows/ci.yml

```diff
@@ -1,37 +1,23 @@
 name: Continuous Integration
 
 on:
-  push:
   pull_request:
 
 jobs:
-  htmlproofer:
-    name: HTMLProofer
+  link-check:
+    name: Link Check
     runs-on: ubuntu-latest
     
     steps:
       - name: Checkout EIP Repository
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
-      
-      - name: Install OpenSSL
-        run: sudo apt-get update && sudo apt-get install -y libcurl4-openssl-dev
-    
-      - name: Install Ruby
-        uses: ruby/setup-ruby@08245253a76fa4d1e459b7809579c62bd9eb718a
-        with:
-          ruby-version: '2.6.0'
-          bundler-cache: true
         
-      - name: Build Website
-        run: |
-          bundle exec jekyll doctor
-          bundle exec jekyll build
```

Comment:
It feels like we should still be building the website *somewhere* in our CI, as we don't want to merge a change that breaks eips.ethereum.org.

Reply:
> We don't want to merge a change that breaks eips.ethereum.org

If a change does get made that breaks `eips.ethereum.org`, then it won't deploy.

Reply:
For PR merging, I think 2 minutes of CI is reasonable.  Shorter is certainly better, but I am pretty apprehensive about removing the build step entirely from CI.

Is there anything we can do to improve the performance of the Jekyll build?  Is there perhaps some caching we could benefit from, or settings we can tweak to improve that?

Reply:
If build performance is the concern, we should probably run a Jekyll `--profile` to see where time is being spent.  Maybe it is something simple/small we can fix.

Reply:
HTMLproofer takes about as long as jekyll.

Reply:
> Is there perhaps some caching we could benefit from

I looked at that earlier, but gave up. I could give it a go again, if you want.

Reply:
While I would appreciate faster CI as much as the next guy, I don't have a strong desire/need for it to warrant thinking anyone should sink a significant amount of time into it.  Another option would be to switch away from Jekyll to something else (with perhaps faster build times).

@lightclient tried to do this a while back, moving to some Rust based system, but I think it didn't support yaml frontmatter which would have required pretty significant changes to every EIP for it to run successfully so we dropped it.  Maybe that tool has added YAML frontmatter support and we could re-evaluate that change?  Or maybe there is another Jekyll-compatible renderer that wouldn't be too hard to hot-swap in?